### DISCUSSION TITLE
What other port forward VPNs are supported by this docker except PIA?

### DISCUSSION THREAD

**Discussion by shawn8888** [haugene/docker-transmission-openvpn#2272] [Q&A] [answered]
Other than PIA, which supports port forward and works great with this docker, are there any other VPNs work too?
I just want to know if there are other options.
Thanks!

**Comment by pkishino** [comment, marked as answer]
hmm,there are a number of them, I know and use Privatevpn

**Reply by shawn8888** [reply]
> 

Thank you! privatevpn looks like a good alternative!

**Reply by jsmith432** [reply]
Is that automatic or does it require any special config for both PIA/privatevpn?

**Reply by shawn8888** [reply]
not sure about privatevpn, but for PIA, you need to enable port forward feature and only some of PIA servers support it.

**Reply by IrishTR** [reply]
> not sure about privatevpn, but for PIA, you need to enable port forward feature and only some of PIA servers support it.

I just switched to PIA from my previous VPN provider VyprVPN(Giganews) and I thought all was working no issues but now I'm seeing a problem where it appears I need to enable port forward?  Any assistance on how to accomplish this?  And is there a list of known working servers for PIA?
